BUILDING.md

```diff
@@ -71,7 +71,7 @@ commands, such as `go test`, should work per package (please see
 A `Makefile` has been provided as a convenience to support repeatable builds.
 Please install the following into `GOPATH` for it to work:
 
-    go get github.com/golang/lint/golint
+    go install github.com/golang/lint/golint
```

Comment:
Same as above for these; also I don't think we still use `golint`, so this is all rather obsolete, but as this branch is mostly in "maintenance", I don't think we should spend much time on rewriting everything, so would suggest reverting the changes and just let them be.

Reply:
reverted.